### PR TITLE
add mindustry-suggestions repository to .github workflow

### PR DESCRIPTION
**Pull request by Arkanic**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
-about: Do not make a new issue for feature requests! Instead, post it on FeatHub, see the README.
+about: Do not make a new issue for feature requests! Instead, post it on the suggestions repository, see the README.
 
 ---
 
-**Do not make a new issue for feature requests!** Instead, post it on FeatHub: https://feathub.com/Anuken/Mindustry
+**Do not make a new issue for feature requests!** Instead, post it on this repository: https://github.com/Anuken/Mindustry-Suggestions
```
Comment (Quezler) [inline .github/ISSUE_TEMPLATE/feature_request.md]: > "this repository"

usage of the word **this** might confuse some people that don't know english very well 🤔

**Comment by DeltaNedas** [comment]
MindustryMods, MindustryBuilds, etc. not Mindustry-*X*
https://github.com/Anuken/Mindustry-Suggestions more like https://github.com/Anuken/MindustrySuggestions

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Arkanic) [reply .github/ISSUE_TEMPLATE/feature_request.md]: oh yes
i suppose it's like desu

**Comment by Anuken** [comment]
The repository isn't ready yet.

**Comment by Arkanic** [comment]
ok

**Pull request closed without merging**